File: 26-09-estruturaDados/nota.c
#include <stdio.h>

int main(){
    float n[3];
    int i;
    float media, soma = 0;
    for(i = 0; i<3;i++){
        printf("Digite a %i nota:", i);
        scanf("%f", &n[i]);
        soma += n[i];
    }
    media = soma/i;
    printf("Media = %.2f \n", media);

    return 0;

}

File: 26-09-estruturaDados/nota.py
from statistics import median


n = []
soma = 0
for i in range(3):
    n += [float(input(f"Digite a nota {i + 1}:"))]
    #nota = (float(input(f"Digite a nota {i + 1}:"))
    #n.append(nota)
    #media = sum(n)/len(n)
    soma += n[i]
media = soma/(i+1)
print(f"Media = {media}")